citadel/install.md

```diff
@@ -7,26 +7,27 @@ Citadel supports the following platforms:
      * Ignition currently only works in headless mode
       (GUI does not render; instead of using `ign gazebo fuel.sdf` command, use
       `ign gazebo -s fuel.sdf` to start the server only).
-
-Windows support is still experimental although most of the packages should work
-as expected. There are no binaries for Windows at this time. The `ign-gazebo`
-package is still not available for Windows; the installation should be done from
-source code. The [Source Installation on Ubuntu Bionic](install_ubuntu_src)
-contains some tips for Windows.
+ * Windows 10
+     * Ignition command line utilities are not yet supported.
+     * All packages up to but not including `ign-gazebo` are currently supported.
+     * DART physics engine is not yet supported.
+     * Qt is not yet supported.
```

Comment:
Humm, the 2nd bullet suggests `ign-gui` is supported, but the 4th suggests it isn't

Reply:
I updated the verbiage here 72f9eba4f94965dd2bfc9b3a1b778344953a5594